### ISSUE TITLE
FYI: Container actions only work on linux

### ISSUE THREAD

**Issue by rjstone** [appleboy/discord-action#3]
Looks like this doesn't work for windows builds. Might want to add a warning to docs.

**Comment by appleboy** [comment]
@rjstone Send the PR.

**Comment by ldd** [comment]
I'm commenting here because someone surely, someday, will come back again and wonder how to do it on windows.

```
      - name: notify
        env:
          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
        run: |
          $MESSAGE=@"
          {\"content\": \"Cool message"}
          "@
          C:\msys64\usr\bin\curl.exe -i -H "Accept: application/json" -H "Content-Type:application/json" -X POST $env:DISCORD_WEBHOOK --data $MESSAGE
```

There you go, no dependencies needed, not even this action. just a simple curl request
Have fun!

**Comment by aar0nsky** [comment]
> 

I am confused, I dont think this solution fits using this github action... Simple curl request doesnt fix the issue of integrating with a github action.

**Comment by appleboy** [comment]
I will handle it.

**Comment by appleboy** [comment]
Only support linux/arm, linux/arm64, linux/amd64 platform.

**Comment by appleboy** [comment]
fix in [d172c11](https://github.com/appleboy/discord-action/commit/d172c11b9f66e85322910c5c7f67c24750782cc3)
